### ISSUE TITLE
Can't click menu items

### ISSUE THREAD

**Issue by ghost** [KSPModStewards/SCANsat#351]
I can open the Projection, Map Type and Celestial Body menus, but nothing happens when I click on the options inside.

**Comment by specs1188** [comment]
I am having the same issue, and now, after re-downloading, I can't even get KSP to load.

**Comment by ensob1** [comment]
same issue 1.8

**Comment by hinneLinks** [comment]
Same Issue here (KSP: 1.8.0.2686 Scansat: v18.13)
I found this in the logs:
```
[EXC 15:11:17.155] MissingMethodException: void UnityEngine.UI.ToggleGroup.SetAllTogglesOff()
	UnityEngine.Events.InvokableCall`1[T1].Invoke (T1 args0) (at <7d9ec060e791409ab3eb85c61e312ed6>:0)
	UnityEngine.Events.UnityEvent`1[T0].Invoke (T0 arg0) (at <7d9ec060e791409ab3eb85c61e312ed6>:0)
	SCANsat.Unity.Unity.SCAN_DropDown.SetElement (System.String element) (at <56120a14b23d4a30b7c79ff112e0a57a>:0)
	SCANsat.Unity.Unity.SCAN_DropDownElement.Select () (at <56120a14b23d4a30b7c79ff112e0a57a>:0)
	UnityEngine.Events.InvokableCall.Invoke () (at <7d9ec060e791409ab3eb85c61e312ed6>:0)
	UnityEngine.Events.UnityEvent.Invoke () (at <7d9ec060e791409ab3eb85c61e312ed6>:0)
	UnityEngine.UI.Button.Press () (at <cf3d7c7e794947019dab4cf03528e6eb>:0)
	UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <cf3d7c7e794947019dab4cf03528e6eb>:0)
	UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <cf3d7c7e794947019dab4cf03528e6eb>:0)
	UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <cf3d7c7e794947019dab4cf03528e6eb>:0)
	UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by DMagic1** [comment]
Should be fixed in version 18.14.
